app/assets/src/components/views/TaxonTreeVis.jsx

```diff
@@ -85,19 +86,13 @@ class TaxonTreeVis extends React.Component {
     }
   }
 
-  static createNode() {
```

Comment:
I just don't think this was used anywhere

Reply:
Good catch...